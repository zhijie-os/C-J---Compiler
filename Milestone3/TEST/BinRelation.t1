main()
{
    boolean b;
    int j;
    b = 3 < false;
}