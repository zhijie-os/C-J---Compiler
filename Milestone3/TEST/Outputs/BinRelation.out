Line 5, binary relation operator < has Non NUMBER type on RHS..
